src/examples/4_head_control.ipynb

```diff
@@ -344,6 +327,56 @@
     "    \n",
     "reachy.turn_off_smoothly()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b478c4e",
+   "metadata": {},
+   "source": [
+    "## Antennas goto"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12b1a208",
+   "metadata": {},
+   "source": [
+    "The antennas can be accessed through the head, using `reachy.head.l_antenna` and `reachy.head.r_antenna`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0af15dc0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(reachy.head.r_antenna)\n",
+    "print(reachy.head.l_antenna)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f45b3140",
+   "metadata": {},
+   "source": [
+    "The methods to control the antennas are quite close to the one available for the head:\n",
+    "- using the `goto` methods, which works like in the [goto_introduction](2_goto_introduction.ipynb) example\n",
+    "- controlling the joint goal positions, namely **reachy.head.r_antenna** and **reachy.head.l_antenna**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e9c33233",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reachy.head.r_antenna.goto(20, duration=0.5)\n",
```

Comment:
Please add a :
reachy.head.turn_on()

Before this cell, otherwise the commands have no effect